README.md

```diff
@@ -243,173 +271,82 @@ The server application can retrieve user details from the authentication certifi
+The `web-eid.js` JavaScript library API specification is available in the project [README](https://github.com/web-eid/web-eid.js).
 
-##### `getCertificate`
+### Web eID authentication token specification
 
-```javascript
-webeid.getCertificate(object options)
-```
-
-| parameter  | type        |                                 |
-|------------|-------------|---------------------------------|
-| `options`  | object      | additional options (_optional_) |
-
-| `options` |                                                        |
-|-----------|--------------------------------------------------------|
-| `filter`  | type of certificate to return. Default is `sign`       |
-| `timeout` | timeout in seconds or `Infinity`. Default is `Infinity`|
+Since there does not exist a standardized format of an authentication proof that fulfills the needs of the Web eID authentication protocol, we use a special purpose format for the Web eID authentication token. We intentionally avoid using the JWT format, but still use its proven basic building blocks: the JSON format and base64-encoding. 
 
-- resolves to an `ArrayBuffer` with the certificate
-- intended to be used with the following `webeid.sign()` operation
-- expected behavior: user is instructed though the process of attaching a reader and a card, if necessary
-- possible reasons for rejection: user cancels certificate selection, no certificates available, some other technical error
-- possible changes: resolving to `undefined` when no certificates available.
-
-##### `sign`
-
-```javascript
-webeid.sign(ArrayBuffer certificate, ArrayBuffer hash, object options)
-```
+The OpenID Connect ID Token JWT format was initially considered, but it was found that any similarities of the Web eID authentication token to the JWT format are actually undesirable, as they would imply that the claims presented in the Web eID authentication token can be trusted and processed while actually they can not be trusted as the token is created in the untrusted user's environment. The solutions are fundamentally different – the purpose of OpenID Connect (and JWT in general) is to exchange identity claims that are signed by a trusted party (usually an authentication server), while the purpose of the Web eID authentication token is to prove that the user is able to create signatures with the private key that corresponds to the presented certificate.
 
-| parameter     | type        |                                   |
-|---------------|-------------|-----------------------------------|
-| `certificate` | ArrayBuffer | certificate to use (**required**) |
-| `hash`        | ArrayBuffer | hash to sign (**required**)       |
-| `options`     | object      | additional options (_optional_)   |
+The full rationale of the Web eID authentication token format design is available in a separate [specification document](https://web-eid.github.io/web-eid-system-architecture-doc/web-eid-auth-token-v2-format-spec.pdf).
```

Comment:
Update the wording and links in the format spec doc - refer to older commit instead of master for OpenID X509 ID Token format spec, use version 1 and version 2 etc.